doc/Developing.md

```diff
@@ -470,6 +487,18 @@ If you already have Xcode installed, but you see the `xcode-select: error: tool
 
 The solution is the same if you have updated to `10.14` (Mojave) and you can see the `gyp: No Xcode or CLT version detected!` error. More details [here](https://github.com/nodejs/node-gyp#on-macos).
 
+### Keytar
+
+See prerequisites for [`node-keytar`](#prerequisites)
+
+On some distributions (e.g. RHEL7, CentOS7) the prebuilt binaries for `node-keytar` might crash at runtime because of some missing shared libraries (libc?)
```

Comment:
Minor: offline discussion to modify the following as it looks incomplete as is:

_Current_: `(libc?)`

_To_: `(ex: libc)` or `(libc, etc.)`